pkg/common/configs/configvalidator.go

```diff
@@ -316,34 +321,46 @@ func checkLimit(limit Limit, currIdx int, userWildCardIdx, groupWildCardIdx *int
 		if name != "*" && !UserRegExp.MatchString(name) {
 			return fmt.Errorf("invalid limit user name '%s' in limit definition", name)
 		}
+
+		if limitValidator.existedUserName[name] {
+			return fmt.Errorf("duplicated user name %s , already existed", name)
+		}
+		limitValidator.existedUserName[name] = true
```

Comment:
Instead of having limitValidator as struct, we can have local variables (say users, groups etc) of type map to do this as anyways removed all other variables in above said struct. This way, we can remove the struct altogether.

Other than this, we are good.

Reply:
Thank you for this good suggestion, addressed in latest PR!